src/Cursor.js

```diff
@@ -69,6 +70,15 @@ var cursorBuildMemoizer = util.memoizeFactory(function (cmp, path, value) {
 var build = cursorBuildMemoizer(function (cmp, path, value) {
   path = path === undefined ? [] : path;
   value = value || util.getRefAtPath(cmp.state, path);
+
+  if (_.isObject(value)) {
```

Comment:
It could better to move this judgement inside Cursor constructor. Then, the subclass of Cursor can get this validation too. (Although there is no such situation, it could better to provide the ability.)